code/game/objects/items/weapons/cigs_lighters.dm

```diff
@@ -197,9 +197,9 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	if(!src.lit)
 		src.lit = 1
 		damtype = "fire"
-		if(reagents.get_reagent_amount("plasma")) // the plasma explodes when exposed to fire
+		if(reagents.get_reagent_amount(PLASMA)) // the plasma explodes when exposed to fire

```

Comment:
inappropriate replacement, this is a reagent ID and those also need to be refactored though